src/layouts/minimal-layout.tsx

```diff
@@ -0,0 +1,38 @@
+import {HEADER_HEIGHT} from '@primer/gatsby-theme-doctocat/src/components/header'
+import PageFooter from '@primer/gatsby-theme-doctocat/src/components/page-footer'
+import TableOfContents from '@primer/gatsby-theme-doctocat/src/components/table-of-contents'
+import {Box, Heading, Text} from '@primer/react'
+import React from 'react'
+import {BaseLayout} from '../components/base-layout'
+
+export default function MinimalLayout({pageContext, children, path}) {
+  const {title, description} = pageContext.frontmatter
+
+  return (
+    <BaseLayout showSidebar={false} title={title} description={description}>
+      <Box sx={{maxWidth: 960, width: '100%', p: [4, 5, 6, 7], mx: 'auto'}}>
```

Comment:
I think we could go a little wider for the `maxWidth`. The text blocks feel a little narrow, especially on wide viewports.